### ISSUE TITLE
ENHANCMENT

### ISSUE THREAD

**Issue by TheOGuser** [kodifitzwell/repo#17]
Good Morning, thank you for what you're doing. in reference to OneMoar, love that you have taken this on, so i am hoping that you can add a feature that POV has.
Single scraping screen, so from the initial click, full screen scrape, with fanart, then it goes straight into the movie or TV Show. At the moment it does not seem smooth.
Hoping you will have a play with this.

**Comment by kodifitzwell** [comment]
thanks for the kind words.
disable duplicates pop up.
what are your thoughts on this?

https://github.com/kodifitzwell/repo/assets/105136042/dc20cbe4-b2b3-4089-b222-1458cbde0c15

**Comment by TheOGuser** [comment]
> thanks for the kind words. disable duplicates pop up. what are your thoughts on this?
> 
>  kodi_dMj3HSl7Kf.mp4

No, thank you 😀
What I was looking for was, pressing the button once then nothing else till it starts playing.
As it is now, you click it, then it flashes a couple times until the program starts.

**Comment by kodifitzwell** [comment]
>Single scraping screen, so from the initial click, full screen scrape, with fanart, then it goes straight into the movie or TV Show.

>pressing the button once then nothing else till it starts playing.

these are not the same.  "full screen scrape" implied you wanted to see the totals/etc.  and the first video is the closest to what POV does,

this is the same "enhancement" with "Icon Only" set.

https://github.com/kodifitzwell/repo/assets/105136042/cfe09874-0305-482b-85fb-87b8a5652773

**Comment by TheOGuser** [comment]
> > Single scraping screen, so from the initial click, full screen scrape, with fanart, then it goes straight into the movie or TV Show.
> 
> > pressing the button once then nothing else till it starts playing.
> 
> these are not the same. "full screen scrape" implied you wanted to see the totals/etc. and the first video is the closest to what POV does,
> 
> this is the same "enhancement" with "Icon Only" set.
> 
>  kodi_2bn5FdbeU3.mp4

Sorry, what would be amazing is... click on the movie once, then would happen till the movie started,  if that makes sense?
I think the umbrella dev tried, when I looked at github bit didn't bother.

**Comment by kodifitzwell** [comment]
https://github.com/kodifitzwell/repo/assets/105136042/06823f0a-8caa-44e0-8ba9-3df41222a338

>i am hoping that you can add a feature that POV has.

POV has nothing like what you asked.

your asks seem king of chaotic, so I wonder if you're asking just to ask.

as this point I'm inclined to remove the "hidden" option all together and suggest that you use the "background" option if you want minimal gui disruption during scrape/resolve.

**Comment by TheOGuser** [comment]
> kodi_OSgVtq1ZaZ.mp4 
> > i am hoping that you can add a feature that POV has.
> 
> POV has nothing like what you asked.
> 
> your asks seem king of chaotic, so I wonder if you're asking just to ask.
> 
> as this point I'm inclined to remove the "hidden" option all together and suggest that you use the "background" option if you want minimal gui disruption during scrape/resolve.

Massive apologies, I am wrong. It's FEN, that has my request.

Can you add this at all? 


https://github.com/kodifitzwell/repo/assets/167613811/d7c6ecd2-4d65-4618-8b20-ca61cbb3beb7

**Comment by kodifitzwell** [comment]
yes, but what is the point in mimic-ing everything Fen?  you're going to use Fen, no?

**Comment by TheOGuser** [comment]
> yes, but what is the point in mimic-ing everything Fen? you're going to use Fen, no?

Not at all, I like OneMoar, and prefer it's settings. Just visually this is nice.

If not, then please close this down. It was just a ask :-)

**Comment by kodifitzwell** [comment]
>I like OneMoar, and prefer it's settings. Just visually this is nice.

fair enough, thanks for the kind words.

if I were to do this I would do it in Dradis first, as it's my fork or Venom.  here is a quick test...

https://github.com/kodifitzwell/repo/assets/105136042/5b9493f4-ec00-43aa-a4ee-dc34dba109be

the spinner might or might not be something I can affect, my connection speed might me slower than yours and/or I think the readfactor on my dev install is default.

**Comment by TheOGuser** [comment]
That was quick.... definitely looking better, for a quick 'knock up'

**Comment by TheOGuser** [comment]
Thanks for the update.
I am super grateful for what you've done.

If at some point you can somehow work it to be like fen, sorry, hope you don't mind me mentioning another addon.

I just think OneMoar could be the go to addon for more soon ;)

**Comment by kodifitzwell** [comment]
https://github.com/kodifitzwell/repo/assets/105136042/03e347df-198e-41e0-af23-7fbe93db2ce5

I believe I have completed this for Dradis.  I am undecided whether I will bring to OneMoar at this time.

**Comment by TheOGuser** [comment]
> kodi_Ay8IUWPcsf.mp4 
> I believe I have completed this for Dradis. I am undecided whether I will bring to OneMoar at this time.

That is awesome. I do like Dradis, as very similar to OG Venom.
Although OneMoar has got a lot of functions which I use more that would stop me going back to Dradis.

Amazing work, so thank you.
If you do ever decide to transfer this over then I would be happy for at least 15 minutes 😀
